frontend/src/App.js

```diff
@@ -83,14 +83,16 @@ const App = () => {
         input:focus {
             outline: 1px solid ${theme.testTheme.linkColor};
         }
-        `;
+    `;
 
     return (
         <ThemeContext.Provider value={theme}>
             <div id="main" css={appStyles}>
                 <Suspense fallback="loading">
                     <Router>
-                        <a className="skip-main" href="#main-content">Skip to main content</a>
+                        <a className="skip-main" href="#main-content">
```

Comment:
Luulen käyttäväni eslinttiä, prettieriä ja editorconfigia mutta näitä tulee aina silti näihin committeihin

Reply:
Mulla tuli kyllä tuo sama eslint, prettier ja editorconfigilla, joten en olisi vielä huolissani :)